src/components/SimplePieChart.js

```diff
@@ -0,0 +1,72 @@
+import * as React from 'react';
```

Comment:
can we import just what we need instead of the whole react/d3 libraries?

Reply:
did this for the react import, but I have to import * from d3, as it has no default exported module (according to this: https://www.freecodecamp.org/news/how-to-get-started-with-d3-and-react-c7da74a5bd9f/)